Services/AuthShibboleth/classes/class.ilAuthShibbolethSettingsGUI.php

```diff
@@ -151,7 +151,7 @@ protected function confirmDeleteRules(): bool
 
         $rule_ids = $this->wrapper->post()->retrieve(
             'rule_ids',
-            $this->refinery->to()->listOf($this->refinery->to()->int())
+            $this->refinery->to()->listOf($this->refinery->kindlyTo()->int())
```

Comment:
What's the reason to change this to kindlyTo()?

Reply:
The reason for changing this to kindlyTo was that I got an error (due to the value being a string) when testing around during refactoring. I thought it would make sense to fix this right away, even when this bug wasn't caused by the refactoring, as it occurred in the component being refactored.

Reply:
@lzehnder thanks for clarification 👍